marlowe-playground-client/src/Home.purs

```diff
@@ -3,42 +3,87 @@ module Home where
 import Prelude hiding (div)
 import Data.Maybe (Maybe(..))
 import Halogen (ClassName(..), ComponentHTML)
-import Halogen.Classes (flex, option1, option2, option3, paddingBottom, vl)
-import Halogen.HTML (a, button, div, h2_, hr_, img, p_, text)
+import Halogen.Classes (arrowLeftDown, arrowLeftUp, arrowRightDown, arrowRightUp, flex, marloweLogo, newProjectBlocklyIcon, newProjectHaskellIcon, newProjectJavascriptIcon, primaryButton, secondaryButton, simulationIcon, simulationIconBlack, vl)
+import Halogen.Css (classNames)
+import Halogen.HTML (a, button, div, h1, h2_, hr_, img, p_, span, span_, text)
 import Halogen.HTML.Events (onClick)
-import Halogen.HTML.Properties (classes, href, src, target)
+import Halogen.HTML.Properties (href, src, target)
 import MainFrame.Types (Action(..), ChildSlots, ModalView(..), State)
+import NewProject.Types as NewProject
+import Projects.Types (Lang(..))
 
 render :: forall m. State -> ComponentHTML Action ChildSlots m
 render state =
-  div [ classes [ ClassName "homepage-container", paddingBottom ] ]
-    [ div [ classes [ ClassName "marlowe-intro-container" ] ]
-        [ div [ classes [ ClassName "text-block" ] ]
-            [ h2_ [ text "What is Marlowe?" ]
-            , p_ [ text "Marlowe is special-purpose language for financial contracts on Cardano, allowing contracts to be written in the language of finance, rather than using a general-purpose language on the blockchain. Because it is special-purpose, it is easier to read, write and understand Marlowe contracts. It is also safer: some sorts of errors are impossible to write, and we can completely analyse contract behaviour without having to run a contract." ]
+  div [ classNames [ "flex", "flex-col", "items-center" ] ]
+    [ h1 [ classNames [ "font-semibold", "text-4xl", "my-16" ] ] [ text "Get started" ]
```

Comment:
```suggestion
  div [ classNames [ "flex", "flex-col", "items-center", "my-16" ] ]
    [ h1 [ classNames [ "font-semibold", "text-4xl", "mb-16" ] ] [ text "Get started" ]
```

if we do this, we get bottom padding equal to the top padding, which looks nicer if the screen isn't tall enough to fit everything comfortably and you have to scroll

Reply:
done